google/cloud/storage/client_default_object_acl_test.cc

```diff
@@ -155,12 +154,12 @@ TEST_F(DefaultObjectAccessControlsTest,
 TEST_F(DefaultObjectAccessControlsTest, DeleteDefaultObjectAcl) {
   EXPECT_CALL(*mock, DeleteDefaultObjectAcl(_))
       .WillOnce(
-          Return(std::make_pair(TransientError(), internal::EmptyResponse{})))
+          Return(StatusOr<internal::EmptyResponse>(TransientError())))
       .WillOnce(Invoke([](internal::DeleteDefaultObjectAclRequest const& r) {
         EXPECT_EQ("test-bucket", r.bucket_name());
         EXPECT_EQ("user-test-user", r.entity());
 
-        return std::make_pair(Status(), internal::EmptyResponse{});
+        return make_status_or( internal::EmptyResponse{});
```

Comment:
Extra space after paren (et seq)?

Reply:
Fixed (here and elsewhere). I guess I trusted clang-format too much.